api/Document.json

```diff
@@ -10240,7 +10260,7 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true,
+              "version_added": false,
```

Comment:
This (`document.tooltipNode`) is also a XUL-only API, like `document.popupNode`. Can you revert this change?